health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/appConfigurationScreen/ImpelComponentWrapper.js

```
@@ -0,0 +1,305 @@
+import React, { Fragment, useEffect, useState } from "react";
+import AppLocalisationWrapper from "./AppLocalisationWrapper";
+
+function ImpelComponentWrapper() {
+  const MODULE_CONSTANTS = "HCM-ADMIN-CONSOLE";
+  const [impelState, setImpelState] = useState(null);
+  const { isLoading: isLoadingAppConfigMdmsData, data: AppConfigMdmsData } = Digit.Hooks.useCustomMDMS(
+    Digit.ULBService.getCurrentTenantId(),
+    MODULE_CONSTANTS,
+    [{ name: "AppScreenConfigTemplateSchema" }, { name: "AppFieldType" }, { name: "DrawerPanelConfig" }, { name: "AppScreenLocalisationConfig" }],
+    {
+      cacheTime: Infinity,
```

Biome (1.9.4)

[error] Use Number.POSITIVE_INFINITY instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.POSITIVE_INFINITY instead.


(lint/style/useNumberNamespace)